e2e/test/scenarios/models/reproductions.cy.spec.ts

```diff
@@ -289,3 +292,67 @@ describe("issue 23103", () => {
     hovercard().findByText("4 distinct values").should("exist");
   });
 });
+
+describe("issue 39150", () => {
+  const ccName = "CC Rating";
+
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it(
+    "shows allows custom columns with the same name in nested models (metabase#39150)",
+    { viewportWidth: 1600 },
```

Comment:
Another way to deal with too many columns on the screen is to specify `fields` in question query and use just the columns you need - there's a small performance benefit to it.

Reply:
I am aware.
But this is less lines of code to read :D

And I'm already limiting results to two rows so we shouldn't have problems with the performance.